src/api/TinRoll.Api/stefanDB.txt

```diff
@@ -0,0 +1 @@
+ options => options.UseSqlServer(@"Server=DESKTOP-P0PO0N5\MSSQLLOCALDB;Database=TinRollDb;Trusted_Connection=True;MultipleActiveResultSets=true"));
```

Comment:
Can you remove this file? Since this is something local to your machine it doesn't really belong in the git repo.